src/features/rainbowkit/WalletAuthModal.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from 'react';
+
+import { ConnectButton } from '@rainbow-me/rainbowkit';
+
+import { useReloadCookieAuth } from 'hooks/useReloadCookieAuth';
+
+export const WalletAuthModal = ({ children }: { children: ReactNode }) => {
+  const { profileId } = useReloadCookieAuth();
+
+  if (!profileId) {
+    return <ConnectButton />;
```

Comment:
this needs some style mojo

Reply:
took a first go at it

Reply:
![Uploading Screenshot 2024-08-21 at 1.22.34 PM.png…]()